controllers/userController.js

```diff
@@ -0,0 +1,70 @@
+const userService = require('../services/userService')
+const { catchAsync } = require('../utils/error')
+
+const signIn = catchAsync(async(req, res) => {
+  const { kakaoAccessToken } = req.body;
+  
+  const kakaoUserInfo = await userService.getKakaoUserInfo(kakaoAccessToken);
+
+  await userService.storeKakaoUserInfo(kakaoUserInfo);
+
+  const token = await userService.generateToken(kakaoUserInfo);
+
+  return res.status(200).json({ 
+    message : 'LOGIN_SUCCESS',
+    accessToken : token
+  });
+});
+
+const getUser = catchAsync(async(req, res) => {
+  const userId = req.user;
+  const userInfo = await userService.getUserById(userId);
+
+  return res.status(200).json({
+    message : 'SUCCESS',
+    data :  userInfo
+  });
+});
+
+const updateUser = catchAsync(async(req, res) => {
+  const userId = req.user;
+  const { hostName, email, phoneNumber, introduction } = req.body;
+
+  await userService.updateUser(hostName, email, phoneNumber, introduction, userId);
+
+  const userInfo = await userService.getUserById(userId);
+
+  return res.status(201).json({
+    message : 'SUCCESS',
+    data : userInfo
+  });
+});
+
+const updateProfileImage = catchAsync(async(req, res) => {
+  const userId = req.user;
+  const profileImageUrl = req.file.location;
+
+  if (!profileImageUrl) {
+    throw new CustomError({
+      status: 401,
+      response: {
+        message: 'Invalid file path'
```

Comment:
반환하는 에러 메시지 대문자로 통일해주셨는데 이 부분도 대문자로 통일해주면 좋을 것 같습니다!

```suggestion
        message: 'INVALID_FILE_PATH'
```

Reply:
네 에러메세지는 대문자로 통일하겠습니다!